source/includes/_endpoint-variable.md

```diff
@@ -28,6 +28,7 @@ name | string | Human-friendly string identifier
 alias | string | More machine-friendly, traditional name for a variable
 description | string | Optional longer string
 id | string | Immutable internal identifier
+notes | string | Optional internal notes for a variables. Same for all users.
```

Comment:
Why is it necessary to say "Same for all users."? So is every other bit of variable metadata here.